### ISSUE TITLE
Setting length

### ISSUE THREAD

**Issue by danielearwicker** [leroydev/no-setting-array-properties-rule#1]
An array’s `length` property can be set, eg. to shorten the array. This should be allowed.

**Comment by leroydev** [comment]
I agree, I think the best way to fix this is to implement a `whitelist` option, in which users can define properties that will be ignored by this rule, like this:
```json
"no-setting-array-properties": {
    "options": {
        "whitelist": ["length"]
    }
}
```

With as default I'd guess: 
```["length", "prototype", "__proto__"]```

What do you think about this? :) @danielearwicker
